sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/ng/SqlBase.g4

```diff
@@ -107,7 +107,8 @@ statement
     | DROP TEMPORARY? FUNCTION (IF EXISTS)? qualifiedName              #dropFunction
     | EXPLAIN explainOption* statement                                 #explain
     | SHOW TABLES ((FROM | IN) db=identifier)?
```

Comment:
@hvanhovell Actually Herman, i tried the following in Hive. 
```SQL
hive> show tables from default;
OK
default__t1_t1_index__
```
should we keep the from option ? There are also some tests which use from option.